students/11th/KimGeonWoo/js/main.js

```diff
@@ -0,0 +1,20 @@
+const input_area = document.querySelector('.input_area');
+const upload_btn = document.querySelector('.upload_btn');
+const comment_section_user_text = document.querySelector('.comment_section_user_text');
+const search_bar_container = document.querySelector('.search_bar_container');
+
+upload_btn.addEventListener('click',function(){
+    let tag_list=document.createElement('li');
+
+    if(input_area.value.length==0){
+        alert("댓글을 입력해주세요");
+    }else{    
+        tag_list.innerHTML=`WeCode `+`${input_area.value}`;
+        comment_section_user_text.appendChild(tag_list);
+        input_area.value="";
+    }
```

Comment:
1. 기왕이면 ===로 비교해주세요
2. 숫자 0의 경우 기본적으로 false로 인식합니다. 이를 이용하면 조건식을 보다 간단하게 적용해볼 수 있겠습니다.
3. 3. 템플릿 리터럴을 사용하시는데 굳이 +로 합쳐줄 필요 없을 것 같습니다
```suggestion
    if(input_area.value.length){
        tag_list.innerHTML = `WeCode ${input_area.value}`;
        comment_section_user_text.appendChild(tag_list);
        input_area.value = "";
    } else {    
        alert("댓글을 입력해주세요");
    }
```